ee/tabby-ui/app/(dashboard)/components/sidebar.tsx

```diff
@@ -31,8 +31,8 @@ export interface SidebarProps {
 }
 
 export default function Sidebar({ children, className }: SidebarProps) {
-  const { data: session } = useSession()
-  const isAdmin = session?.isAdmin || false
+  const [{ data }] = useMe()
+  const isAdmin = data?.me?.isAdmin
```

Comment:
```suggestion
  const isAdmin = data?.me.isAdmin
```

`me` is always not null if data is not null?